data/tests.yml

```diff
@@ -271,6 +271,36 @@
           - style: solid_line
             color: white
 
+### Mis-tagged roads
+
+- way_id: 240294912
+  description: Invalid lanes:both_ways value
+  # TODO Enable after https://github.com/a-b-street/osm2lanes/pull/168
+  rust: false
+  tags:
+    highway: "primary"
+    lanes: "4"
+    lanes:both_ways: "2"
+    sidewalk: "both"
+  driving_side: left
+  ISO 3166-2: GB-NET
+  road:
+    highway: primary
+    lanes:
```

Comment:
I am actually not sure what the correct interpretation of these tags is...

Reply:
maybe we need some kind of "don't care" output? just to check that we don't panic

likewise I intend to and an error checking possibility in the output

Reply:
That'd be nice, will keep it in mind. I enabled the test and made the expectations match the output -- which is kind of half-reasonable in this case. Just ignore the bogus both_ways value.